pkg/cnb/create_builder.go

```diff
@@ -50,6 +45,10 @@ func (r *RemoteBuilderCreator) CreateBuilder(
 	if err != nil {
 		return buildapi.BuilderRecord{}, err
 	}
+	lifecycleImage, _, err := r.RegistryClient.Fetch(lifecycleKeychain, clusterLifecycle.Spec.Image) // TODO: should there be a "latest image" on the status?
```

Comment:
That way we could at least know the exact digest that was used, even if we don't use it anywhere?